measure_extinction/extdata.py

```diff
@@ -98,7 +97,7 @@ def _get_column_val(column):
         return float(column)
 
 
-def AverageExtData(extdatas):
+def AverageExtData(extdatas, min_number=1):
```

Comment:
Thinking that the default should be larger than 1.  Maybe 3 or 5?  Taking an average of 1 curve doesn't sound like an average.  And given the nice warning you added later, seems like it would be better to have the default be higher and then the user would have to change it if a different value is desired.

Reply:
Ok, I changed the default to 3.